tests/system/action/poll/test_update.py

```diff
@@ -130,3 +138,230 @@ def test_not_allowed_for_non_analog(self) -> None:
             "Following options are not allowed in this state and type: "
             "votesvalid, votesinvalid, votescast"
         ) in response.data.decode()
+
+
+class UpdatePollTestCase(BaseActionTestCase):
```

Comment:
Same as above

Reply:
Merged the tests into one class. Removed some surplus tests.